Linux Script/configure-linux.sh

```diff
@@ -215,18 +218,23 @@ checkIfSupportedOS()
 	case "$LINUX_DIST_IN_LOWER_CASE" in
 		*"ubuntu"* )
 		echo "INFO: Operating system is Ubuntu."
+		PKG_MGR="apt-get"
 		;;
-		*"redhat"* )
+		*"red"* )
 		echo "INFO: Operating system is Red Hat."
+		PKG_MGR="yum"
```

Comment:
@mostlyjason, Usually, yum and apt-get package managers are already present on the respective systems. For safer side, I am working on to implement conditions to check for the presence of yum and apt-get commands then I will set the `PKG_MGR` accordingly. If `yum` or `apt-get` would not already present in the system then I will show an error message to install `yum` or `apt-get` and then exit the script execution.